src/integration-test/resources/HistoryIT/__files/test8_readAttachmentNotFound/response.json

```diff
@@ -0,0 +1,5 @@
+{
+	"title": "Not Found",
+	"status": 404,
+	"detail": "No history found for message id 67db7260-5829-40f4-ab3d-0ff55d117fb0"
```

Comment:
Shouldn't it say something like "attachment doesNotExist.jpg is not found for message id..."? I.e. history should/might be present, but no image with name doesNotExist.jpg should be found in history, right?

Reply:
If there is no history with a given message id this is the response.